metadata.yaml

```diff
@@ -22,5 +22,10 @@ resources:
     type: oci-image
     description: Spring Boot application image.
 
+requires:
+  ingress:
```

Comment:
Is the ingress optional?

Reply:
It is, but the "optional" field in metadata seems to be informational only

Reply:
Is it sill worth adding so that it is explicit?

Reply:
Won't hurt

Reply:
Updated, thanks!